src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -216,6 +216,8 @@ function MoneyRequestPreview(props) {
             message += ` • ${props.translate('iou.approved')}`;
         } else if (props.iouReport.isWaitingOnBankAccount) {
             message += ` • ${props.translate('iou.pending')}`;
+        } else if (props.action.originalMessage && props.action.originalMessage.reason === CONST.IOU.CANCEL_REASON.PAYMENT_EXPIRED) {
```

Comment:
```suggestion
        } else if (props.action.originalMessage && props.action.originalMessage.cancellationReason === CONST.IOU.CANCEL_REASON.PAYMENT_EXPIRED) {
```

Reply:
Ah, my previous mock data is a bit outdated (we're working on the backend changes for this currently).

<details> 
<summary> updated DEQUEUED report action </summary>

```json
      "2084950335095234642": {
        "person": [
          {
            "type": "TEXT",
            "style": "strong",
            "text": "test9@kycwallet.com"
          }
        ],
        "actorAccountID": 9422,
        "message": [
          {
            "type": "TEXT",
            "style": "strong",
            "text": "You"
          },
          {
            "type": "TEXT",
            "style": "normal",
            "text": " cancelled the reimbursement"
          }
        ],
        "originalMessage": {
          "lastModified": "2023-11-07 10:57:32.041",
          "cancellationReason": "CANCEL_REASON_PAYMENT_EXPIRED"
        },
        "avatar": "https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/default-avatar_17.png",
        "created": "2023-11-07 10:57:32.041",
        "timestamp": 1699354652,
        "reportActionTimestamp": 1699354652041,
        "automatic": false,
        "actionName": "REIMBURSEMENTDEQUEUED",
        "shouldShow": true,
        "reportActionID": "2084950335095234642",
        "previousReportActionID": "6468451880113722900",
        "lastModified": "2023-11-07 10:57:32.041",
        "whisperedToAccountIDs": []
      }
    }
```

</details>